text/java/0000-clojure-tools.md

```diff
@@ -0,0 +1,51 @@
+# Clojure Tools Buildpack
+
+## Summary
+
+A Clojure Tools buildpack for building Clojure applications using [deps][d] and [tools build][t] has been created [by the community][b].  It is based on the same technologies and is nearly indistinguishable from the other Paketo Buildpacks, especially the JVM build-system buildpacks, and it should be adopted into the project.
+
+[d]: https://clojure.org/guides/deps_and_cli
+[t]: https://clojure.org/guides/tools_build
+[b]: https://github.com/eddumelendez/clojure
+
+## Motivation
+
+This is an additional buildpack, for building JVM-based applications based on Clojure, and offers yet another workload that can be built by Paketo buildpacks.
+
+## Detailed Explanation
+
+The Clojure Tools Buildpack will be moved into the paketo-buildpacks Github organization and added to the Java sub-team. Maintenance for this buildpack will be provided by the Java subteam.
+
+The following changes will be made:
+
+* [Clojure Tools Buildpack][b] moved to Paketo Buildpacks org
+* Buildpack will have paketo-buildpacks/clojure-tools ID
+* Buildpack will be published to gcr.io/paketo-buildpacks/clojure-tools
+* Buildpack will have go module of github.com/paketo-buildpacks/clojure-tools
+* Github Actions will be added through [pipeline-builder][p]
+* An action will be added to [pipeline-builder][p] with the purpose of checking for new versions of [Clojure Tools][c]
+
```

Comment:
@eddumelendez Can you submit an example order group for the [paketo-buildpacks/java composite buildpack](https://github.com/paketo-buildpacks/java/blob/main/buildpack.toml#L30-L57)?

Where does this buildpack need to fit? I'm guessing in a spot similar to maven/gradle, but would like to confirm. Thanks

Reply:
between leiningen and gradle would be a good fit

```
{ id = "paketo-buildpacks/leiningen",                  version="3.2.2", optional = true },
{ id = "paketo-buildpacks/clojure-tools",            version="1.0.0", optional = true },
{ id = "paketo-buildpacks/gradle",                      version="5.4.0", optional = true },
```

Reply:
OK, sounds good to me.